test/e2e/e2e_ginkgo/csv_e2eg_test.go

```diff
@@ -0,0 +1,147 @@
+package e2e_ginkgo
+
+import (
+	"fmt"
+
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/client/clientset/versioned"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/lib/operatorclient"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+
+	o "github.com/onsi/gomega"
+)
+
+type cleanupFunc func()
```

Comment:
All the definitions in this _test.go file won't be usable by other files. Please consider changing the file name to _util.go

Reply:
Done. I have restructured the code now to have separate util pkg for all the helper files